lib/webpacker/railtie.rb

```diff
@@ -15,4 +15,10 @@ class Webpacker::Engine < ::Rails::Engine
     Webpacker.bootstrap
     Spring.after_fork { Webpacker.bootstrap } if defined?(Spring)
   end
+
+  initializer "webpacker.logger" do
+    config.after_initialize do |app|
+      Webpacker.logger = ::Rails.logger
```

Comment:
I'd suppose this errs on the side of making sure the library's logger stays in sync with `Rails.logger` since it's more likely that `Rails.logger` would get overridden in app configuration.